build.cmd

```diff
@@ -6,10 +6,18 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly.
 
+:ReadArguments
+:: Read in the args to determine whether to run the native build, managed build, or both (default)
+set "__args=%*"
+if /i [%1] == [native]    	(set __buildSpec=native&&set "__args=%__args:~6%"&&shift&&goto Tools)
+if /i [%1] == [managed]    	(set __buildSpec=managed&&set "__args=%__args:~7%"&&shift&&goto Tools)
```

Comment:
It's tabbed so that the parens for native and managed are in line. It's not really necessary though since there are only two items.